### PR TITLE
Update to Java 17

### PR DESCRIPTION
**Pull request by willmostly**
This updates the source and target version to java 17. This requires adding VM options for the surefire plugin and `org.javalite:activejdbc-instrumentation`. A `mvnw` is added to facilitate passing the required option to `activejdbc-instrumentation`.

### CODE REVIEW

**Comment by mosabua** [comment]
Also .. @vishalya .. do we need to get your PRs in first?

File: pom.xml [hunk 1]
```diff
@@ -12,8 +12,8 @@
     <version>1.9.5</version>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
```
Comment (mosabua) [inline pom.xml:15]: Not sure we want to do this now already .. we could just make it work with 17 to build .. and then switch language level later

Comment (willmostly) [reply pom.xml:15]: Sounds good. I closed this and will add the build options in once the maven wrapper PR https://github.com/trinodb/trino-gateway/pull/8 is merged. I don't think this should conflict with Vishal's package renaming.

Comment (vishalya) [reply pom.xml:15]: Yes, that makes sense. I will have 2 more PR to bring our code in sync with this repo and then we can go on enhancing the rest.

**Pull request closed without merging**